packages/search/index.test.js

```diff
@@ -9,10 +9,6 @@ import Search, {
 import { HitsComponent } from './hits'
 import { indexContent } from './tools'
 
-const renderWithProvider = (ui) => {
```

Comment:
I removed this helper, as I wanted to configure `<SearchProvider />` with props in specific tests.